blis/about.py

```diff
@@ -5,7 +5,7 @@
 # https://github.com/pypa/warehouse/blob/master/warehouse/__about__.py
 
 __name__ = "blis"
-__version__ = "0.9.0"
+__version__ = "0.9.1.dev1"
```

Comment:
It doesn't actually matter, but this confused me and made me wonder if I'd missed a release because we usually start with `dev0`.

```suggestion
__version__ = "0.9.1.dev0"
```